### PR TITLE
Update git workflow resources

### LINKED ISSUE #372: Add more resources for how to use branches

### PR DESCRIPTION
**Pull request by maelle**
Fix #372

Thanks @statnmap @ha0ye for links to resources.

### CODE REVIEW

File: maintenance_collaboration.Rmd [hunk 1]
```diff
@@ -93,13 +93,19 @@ If your problem is _recruiting_ collaborators, you can post an open call like Ji
 
 ### Working with collaborators (including yourself) {#gitflow}
 
-You could implement the "[gitflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow)" philosophy as explained by Amanda Dobbyn in [this blog post](https://ropensci.org/blog/2018/04/20/monkeydo/#gitflow). In a nutshell,
+[Branches](https://happygitwithr.com/git-branches.html) are cheap. Use them extensively when developing features, testing out new ideas, fixing problems.
 
-* Keep the default branch (main or master) as shippable as much as possible
+One of the branches is the default / main branch, where, if you follow [trunk-based development](https://www.atlassian.com/continuous-delivery/continuous-integration/trunk-based-development), you "merge small, frequent updates".
+See also [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow) and [GitLab flow](https://docs.gitlab.com/ee/topics/gitlab_flow.html) docs.
+You might want to pair the frequent updating with frequent version numbers incrementing (in `DESCRIPTION`).
```
Comment (mpadge) [inline maintenance_collaboration.Rmd:100]: "frequent version numbers increasing" -> "frequent version number incrementing" (singular) OR "frequent incrementing of version numbers" (correct English)

File: maintenance_collaboration.Rmd [hunk 2]
```diff
@@ -93,13 +93,19 @@ If your problem is _recruiting_ collaborators, you can post an open call like Ji
...
 
-* [Branches](https://happygitwithr.com/git-branches.html) are cheap. Use them extensively when developing features, testing out new ideas, fixing problems.
+One particular aspect of working with collaborators is reviewing pull requests. 
+Some useful guidance:
```
Comment (mpadge) [inline maintenance_collaboration.Rmd:103]: That dot-point should maybe be joined on to the prior one: "reviewing pull requests, with some useful guidance at:" (or something like that)

**Review by mpadge — CHANGES_REQUESTED** [review]

**Review by mpadge — APPROVED** [review]
ah, that's better. Nice!

**Comment by maelle** [comment]
Thank you @mpadge

**Pull request merged**